src/send/saga.test.ts

```diff
@@ -63,6 +63,23 @@ jest.mock('src/transactions/types', () => {
   }
 })
 
+jest.mock('src/web3/networkConfig', () => {
+  const originalModule = jest.requireActual('src/web3/networkConfig')
+  return {
+    __esModule: true,
+    ...originalModule,
+    default: {
+      ...originalModule.default,
+      networkToNetworkId: {
+        celo: 'celo-alfajores',
+        ethereum: 'ethereuim-sepolia',
+      },
+      supportedNetworkIds: ['celo-alfajores', 'ethereum-sepolia'],
```

Comment:
is this needed anymore? looks like you removed it from networkConfig.ts

Reply:
good catch, ty